windows/build.js

```diff
@@ -0,0 +1,84 @@
+var fs = require('fs');
+var path = require('path');
+var exec = require('child_process').exec;
+var argv = require('yargs').argv;
+var Promise = require('bluebird');
+
+var nwappManifest = require('../nwapp/package.json');
+var appVersion = nwappManifest.version;
+
+var stat = Promise.promisify(fs.stat);
+
+var possibleProgramFilePaths = [
+  'C:\\Program Files (x86)',
+  'C:\\Program Files'
+];
+
+var possibleSignToolPaths = [
+  'Microsoft SDKs\\Windows\\v7.1\\Bin\\signtool.exe',
+  'Microsoft SDKs\\Windows\\v7.1A\\Bin\\signtool.exe',
+  'Windows Kits\\8.1\\bin\\x64\\signtool.exe',
+  'Windows Kits\\8.1\\bin\\x86\\signtool.exe',
+  'Windows Kits\\8.0\\bin\\x64\\signtool.exe',
+  'Windows Kits\\8.0\\bin\\x86\\signtool.exe'
+];
+
+var statResults = possibleProgramFilePaths.reduce(function(statResults, programFilePath) {
+  return statResults.concat(possibleSignToolPaths.map(function(signToolPath) {
+    var fullSignToolPath = path.join(programFilePath, signToolPath);
+    return stat(fullSignToolPath)
+      .then(function() {
+        return fullSignToolPath;
+      })
+  }));
+}, []);
+
+
+Promise.any(statResults)
+  .then(function(signToolPath) {
+    var certPassword = argv.password || argv.p || '';
+
+    var commandList = [
+      '"' + signToolPath + '" sign /f "' + path.resolve(__dirname, '..\\certificates\\troupe-cert.pfx') + '" /p ' + certPassword + ' "' + path.resolve(__dirname, '..\\opt\\Gitter\\win32\\Gitter.exe') + '"',
+      '"C:\\Program Files (x86)\\Inno Setup 5\\ISCC.exe" "' + path.resolve(__dirname, 'gitter.iss') + '"',
```

Comment:
@trevorah I think you misunderstood this part of the code? We look for the `signtool.exe` on your PC in a variety of locations(which it can be). When any of those checks succeed, `Promise.any(statResults)`, we use the path found to run the build commands.